lib/command_runner.dart

```diff
@@ -147,6 +154,21 @@ class CommandRunner<T> {
     Command? command;
     var commandString = executableName;
 
+    // Returns commands similar to `name`, in sorted order.
```

Comment:
Does this need to be a local function? Can you make it a private method?

Reply:
It does read state from the surrounding scope. I could instead pass that as a parameter, but the local method makes it a bit cleaner and it isn't used outside this method?

Reply:
If it were me, I would pass in the stuff as a parameter. This is quite long for a local function.